src/open_inwoner/components/templates/components/File/File.html

```diff
@@ -49,8 +49,8 @@
                     </div>
                 {% enddropdown %}
             {% elif show_download %}
-                {% trans "Download" as download %}
-                {% link href=url text=download primary=True download=True hide_text=True extra_classes="file__download" icon="download" icon_outlined=True icon_position="before" %}
+                {% trans "Download " as download %}
```

Comment:
```suggestion
                {% trans "Download" as download %}
```

Reply:
I would avoid mixing translations with formatting and instead call the filter twice.